README.md

```diff
@@ -14,8 +14,35 @@ Documentation about Azure Spring Apps is available [here](https://learn.microsof
 
 ---
 
-## 🚀 How to get started
-
+## 🚀 How to generate a project
+### CLI
 1. Install the package with `npm i generator-jhipster-azure-spring-apps`
 1. Create and navigate to a directory
 1. Generate the application with `jhipster-azure-spring-apps createtodoapp`
+
+### JHipster Online
+Go to [Jhipster Online](https://start.jhipster.tech/generate-azure-application) and generate your application.
+
+## 🎉 How to deploy on Azure
+1. Install AZD, you can try one of the following options:
```

Comment:
I would suggest we use the official link for azd installation https://learn.microsoft.com/en-us/azure/developer/azure-developer-cli/install-azd?tabs=winget-windows%2Cbrew-mac%2Cscript-linux&pivots=os-windows which covers all os and is clear and simple to follow. The following content only applies to windows.